### PR TITLE
UHM-4354 Move Location, Location Tag, and Lcation Attribute Types installation out of PIH core and into config repos

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/UHM-4354

### CODE REVIEW

File: configuration/locations/core-locations.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+UUID,Void/Retire,Name,Description
+AF939782-898B-409A-99A4-D1653484EDBD,,Patient Home,The patient is visited at home
```
Comment (mogoodrich) [inline configuration/locations/core-locations.csv:2]: What about "Unknown Location"? Are we assuming that is part of the core OpenMRS install?

Comment (mseaton) [reply configuration/locations/core-locations.csv:2]: I would assume so, but doesn't hurt to include it for completeness.  I'd vote to include it here.

**Review by mseaton — APPROVED** [review]
I did not review the details of the csvs - hopefully you got all of those right and got them by exporting from an existing DB, as that leaves less room for transcription errors.

File: configuration/locations/core-locations.csv (hunk 1 above)
Comment (brandones) [reply configuration/locations/core-locations.csv:2]: Initially I had it included and it caused an NPE in concept validation, something about its location attributes being null. Chose not to worry about it.

**Comment by brandones** [comment]
I've been using vim, since all this stuff is in code already. Just a few `g/` and `%s/` commands to clean the MDD version into a valid CSV.

File: configuration/locations/core-locations.csv (hunk 1 above)
Comment (mogoodrich) [reply configuration/locations/core-locations.csv:2]: Strange... what was the error @brandones

Comment (brandones) [reply configuration/locations/core-locations.csv:2]: Huh, it worked this time when I re-ran it. Added that line back. We'll see if it causes trouble.

**Pull request merged**